Northstar.CustomServers/mod/scripts/vscripts/titan/_replacement_titans.gnut

```diff
@@ -524,10 +525,48 @@ function req()
 
 bool function ClientCommand_RequestTitan( entity player, array<string> args )
 {
+	if( GAMETYPE == FORT_WAR )
```

Comment:
dont add gamemode specific stuff to these files make it a callback and set it in the gamemode file